src/main/webapp/app/shared/metis/message/message-inline-input/message-inline-input.component.ts

```
@@ -75,4 +98,84 @@ export class MessageInlineInputComponent extends PostingCreateEditDirective<Post
         this.localStorageService.store('chatWarningDismissed', true);
     }
+
+    /**
+     * Helper method to check if iris is activated in the settings
+     * @param channelDTO channel to be checked for
+     * @private
+     */
+    private checkIrisSettings(channelDTO?: ChannelDTO): void {
+        switch (channelDTO?.subType) {
+            case ChannelSubType.GENERAL:
+                const course = this.course();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)